content/hi/docs/reference/glossary/downward-api.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Downward API
```

Comment:
You can use Hindi here if you prefer; if you do, you should use `aka` to also show the English term.